Example/Emission/AppDelegate.m

```diff
@@ -10,7 +10,7 @@
 #import <React/RCTUtils.h>
 #import <TargetConditionals.h>
 
-#define ARTIST @"banksy"
+#define ARTIST @"alex-katz"
```

Comment:
Forgot to change this? No prob, we can switch it up from Banksy from time to time!